notebooks/community/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<h3>Deploy the model to an endpoint -&gt; Deploy the Model resource to an Endpoint resource</h3><h3><br></h3><h3>Rephase <code>BigQuery ML is not suitable for real-time predictions</code> from a negative.  Try something like: it is recommend to use Vertex AI online prediction for low latency real-time prediction.</h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>